common/common_test.go

```diff
@@ -950,3 +953,71 @@ func TestTimeFromUnixTimestampFloat(t *testing.T) {
 		t.Error("Test failed. Common TimeFromUnixTimestampFloat. Converted invalid syntax.")
 	}
 }
+
+func TestGetDefaultDataDir(t *testing.T) {
+	expectedOutput := os.Getenv("APPDATA") + GetOSPathSlash() + "GoCryptoTrader"
+	actualOutput := GetDefaultDataDir("windows")
+	t.Log(actualOutput)
+	if actualOutput != expectedOutput {
+		t.Errorf("Unexpected result. Got: %v Expected: %v", actualOutput, expectedOutput)
+	}
+}
+
+func TestCreateDir(t *testing.T) {
+	switch runtime.GOOS {
+	case "windows":
+		// test for a directory that exists
+		dir := "c:\\temp"
```

Comment:
Would be better to use env TEMP as this current value is an old default used by Windows pre Windows XP but does not exist on Windows 10 (Instead it's located in %USER%\AppData\Local\Temp)